### DISCUSSION TITLE
What is the font used in this screenshot?

### DISCUSSION THREAD

**Discussion by ghost** [marcofugaro/threejs-modern-app#12] [Q&A] [answered]
Hello @marcofugaro,

as I found this beautiful three.js template, I came across a terminal screenshot of yours and was wondering what the name of that font is.
It is the one found in the [README.md](https://github.com/marcofugaro/threejs-modern-app#features):

![](https://github.com/marcofugaro/threejs-modern-app/blob/master/.github/screenshots/console.png?raw=true)

I appreciate your excellent programming font taste.
I reckon it is a very well known one (it looks familiar to me) but I've asked in many places and nobody was able to give me an answer. Do you mind telling me the name?

**Comment by jefftheduck** [comment]
Idk honestly

**Comment by marcofugaro** [comment, marked as answer]
hey, the font I use is ProFontWindows 🙂 you can get it here:

https://www.fontsquirrel.com/fonts/profontwindows
